docs/source/reference/services.md

```diff
@@ -86,10 +86,18 @@ Hub-Managed Service would include:
 This example would be configured as follows in `jupyterhub_config.py`:
 
 ```python
+c.JupyterHub.load_roles = [
+    {
+        "name": "idle-culler",
+        "scopes": [
+            "users:servers",
```

Comment:
`idle-culler` also needs access to users' last activity which is currently not covered by the `users:servers` scope (it does include `read:users:name` scope though). Shall we make the `read:users:activity` a subscope of `users:servers`? Or just add it to the role definition?

Reply:
I'll add the role definition. Good catch!